packages/vscode-integration/src/glsp-webview.ts

```diff
@@ -157,19 +162,17 @@ export class GLSPWebView extends Disposable implements ExtensionActionDispatcher
     }
 
     protected async sendToWebview(message: any): Promise<void> {
-        if (isActionMessage(message) || isDiagramIdentifier(message) || isResponseMessage(message)) {
```

Comment:
I have an RFC here: I removed this check as it seemed superfluous to me.  
  
While testing, all messages I could log fell into one of the three categories. If anybody knows if the check was put here for a reason I will restore the line, otherwise I don't believe it is needed.

Reply:
This check ensures that only "valid" objects, i.e. objects that can be handled by the webview, are sent to the webview. Hopefull all messages that are send directly by one of our components fall into this method, however, we cannot make such assumptions for client customizations. 

If you want to omit this check we should probably refine the type of the message object. e.g.

``` ts
export type GLSPWebviewMessage= ActionMessage| SprottyDiagramIdentifer | ResponseMessage ;

protected async sendToWebview(message: GLSPWebviewMessage): Promise<void> {
...
}
```

Reply:
Ah I understand now. Thank you! :)

I believe that it's best if we don't interfere with messages from the server here and just send them over to the webview unchecked. If we catch any messages here we might completely prevent extension authors from having custom implementations where they need to send unusual or "invalid" data from the server to the webview.

To mitigate the issue where "invalid" objects are propagated from the server to the client, we could in the future expose an entry point for extension authors to define action handlers similar to how we have it already. There they could catch their custom events and prevent them to be propagated further.

If it's ok with you I will adjust the typing and method signaure to only accept valid objects in this function. I would be happy to hear your view on this.